src/tests/test_pkcs11_high_level.cpp

```diff
@@ -850,10 +854,11 @@ Test::Result test_rsa_sign_verify()
 
       Botan::PK_Verifier verifier(keypair.first, emsa, Botan::IEEE_1363, "pkcs11");
       bool rsa_ok = false;
-      if ( multipart )
+      if(multipart)
          {
          verifier.update(plaintext.data(), plaintext.size() / 2);
-         rsa_ok = verifier.verify_message(plaintext.data() + plaintext.size() / 2, plaintext.size() / 2, signature.data(), signature.size());
+         rsa_ok = verifier.verify_message(plaintext.data() + plaintext.size() / 2, plaintext.size() / 2, signature.data(),
+         signature.size());
```

Comment:
Wrong indent, also if a break is required of a long line break it in the middle rather than lopping off a tiny bit of the end, say like

```
rsa_ok = verifier.verify_message(plaintext.data() + plaintext.size() / 2, plaintext.size() / 2, 
                                 signature.data(), signature.size());
```